platforms/org.eclipse.mita.platform.arduinouno/ARD_SDK/ARD_Timer.h

```diff
@@ -0,0 +1,20 @@
+/*
+ * ARD_Timer.h
+ *
+ *  Created on: 13.03.2018
+ *      Author: administrator
+ */
+
+#ifndef SRC_GEN_BASE_ARD_TIMER_H_
+#define SRC_GEN_BASE_ARD_TIMER_H_
+
+#include "ARD_Retcode.h"
+#include "MitaTime.h"
+#include "avr/interrupt.h"
```

Comment:
This include should be in `ARD_Timer.c` if you need it. In this header you make no use of anything defined in `interrupt.h`